src/app/CommandHandler.h

```diff
@@ -190,9 +226,73 @@ class CommandHandler : public Messaging::ExchangeDelegate
     CHIP_ERROR AddClusterSpecificFailure(const ConcreteCommandPath & aCommandPath, ClusterStatus aClusterStatus);
 
     Protocols::InteractionModel::Status ProcessInvokeRequest(System::PacketBufferHandle && payload, bool isTimedInvoke);
-    CHIP_ERROR PrepareCommand(const ConcreteCommandPath & aCommandPath, bool aStartDataStruct = true);
+
+    /**
+     * This adds a new CommandDataIB element into InvokeResponses for the associated
+     * aRequestCommandPath. This adds up until the `CommandFields` element within
+     * `CommandDataIB`.
```

Comment:
Does this start the CommandFields  struct?  That depends on the arguments.  Should be documented, since the member of InvokeResponseParameters is not documented...

Reply:
Documented it here and in `InvokeResponseParameters`